WrathCombo/Combos/PvE/DRK/DRK_Config.cs

```diff
@@ -176,7 +176,7 @@ internal static void Draw(CustomComboPreset preset)
 
                 #region PVP
 
-                case CustomComboPreset.DRKPvP_Burst:
+                case CustomComboPreset.DRKPvP_Shadowbringer:
```

Comment:
Shadowbringer requires a minimum of 12,000 HP to use (or Dark Arts to be present).
The logic will softlock if your HP drops below that, so you need a minimum HP check.

Reply:
Makes sense will implement